homework/03LessonThree.js

```diff
@@ -80,7 +83,10 @@ var problemSevenAnswer;
 //
 // Your Answer Goes Here:
 //
-//
+//The ternary operator (?) is like a toggle. It outputs one result and then
```

Comment:
Ternaries are for more than toggling! Let's update this answer with a little more generic description

Read this! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator